spec/support/test_helpers/provider_user_permissions_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ProviderUserPermissionsHelper
+  def permit_make_decisions(dfe_sign_in_uid: 'DFE_SIGN_IN_UID', provider: nil)
```

Comment:
Nice!
- should it have a bang, as we're side-effecting?
- would be nice to include see_safeguarding_information here? Maybe a follow up refactor.